scriptmodules/supplementary/skyscraper.sh

```diff
@@ -199,44 +205,33 @@ function configure_skyscraper() {
 function _init_config_skyscraper() {
     local scraper_conf_dir="$configdir/all/skyscraper"
 
-    # Make sure the `artwork.xml` and other conf file(s) are present, but don't overwrite them on upgrades
-    local f_conf
-    for f_conf in artwork.xml aliasMap.csv; do
-        if [[ -f "$scraper_conf_dir/$f_conf" ]]; then
-            cp -f "$md_inst/$f_conf" "$scraper_conf_dir/$f_conf.default"
-        else
-            cp "$md_inst/$f_conf" "$scraper_conf_dir"
-        fi
-    done
-
     # If we don't have a previous config.ini file, copy the example one
-    if [[ ! -f "$scraper_conf_dir/config.ini" ]]; then
-        cp "$md_inst/config.ini.example" "$scraper_conf_dir/config.ini"
-        sed -i 's/\[esgamelist\]/[esgamelist]\ncacheScreenshots="false"/' "$scraper_conf_dir/config.ini"
-    fi
+    [[ ! -f "$scraper_conf_dir/config.ini" ]] && cp "$md_inst/config.ini.example" "$scraper_conf_dir/config.ini"
 
     # Try to find the rest of the necessary files from the qmake build file
-    # They should be listed in the `unix:examples.file` configuration line
-    if [[ $(grep unix:examples.files "$md_build/skyscraper.pro" 2>/dev/null | cut -d= -f2-) ]]; then
-        local files=$(grep unix:examples.files "$md_build/skyscraper.pro" | cut -d= -f2-)
-        local file
-
-        for file in $files; do
-            # Copy the files to the configuration folder. Skip config.ini, artwork.xml and aliasMap.csv
-            if [[ $file != "artwork.xml" && $file != "config.ini" && $file != "aliasMap.csv" ]]; then
-                cp -f "$md_build/$file" "$scraper_conf_dir"
-            fi
-        done
-    else
-        # Fallback to the known resource files list
-        cp -f "$md_inst/artwork.xml.example"* "$scraper_conf_dir"
-
-        # Copy resources and readme
-        local resource_file
-        for resource_file in README.md mameMap.csv tgdb_developers.json tgdb_publishers.json hints.txt; do
-            cp -f "$md_inst/$resource_file" "$scraper_conf_dir"
-        done
+    # They should be listed in the `unix:examples.files` configuration line
+    iniConfig "=" "" "$md_build/skyscraper.pro"
+    iniGet "unix:examples.files"
+    local files="$ini_value"
+    if [[ -z "$files" ]]; then
+        files="README.md hints.xml artwork.xml artwork.xml.example1 artwork.xml.example2"
```

Comment:
documentation files don't need to be copied to the configuration folder. Remove `README.md` and `docs/ARTWORK.md`.

Reply:
(They'll need to be removed from **unix:examples.files** too if the binary/source installs are to remain in sync.)

Reply:
Adding them to the line of `config.ini)` should be fine.

Reply:
They should be added at all - the README.md can be added in the `install_` files and `ARTWORK.md` is already part of the `docs`, which is also copied to the install folder.

Reply:
> They should be added at all

We are talking about "adding" them to [this line](https://github.com/RetroPie/RetroPie-Setup/blob/b05af36837499a345312a5226cd3e9441a6709b6/scriptmodules/supplementary/skyscraper.sh#L222) in the `case` statement so that they will *not* be copied to the config dir, even though present in the **unix:examples.files** value.

Reply:
Sorry, I meant to **not** add them to be copied.